### PR TITLE
fix: Eslint warnings

### PR DESCRIPTION
**Pull request by liiil825**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -68,7 +68,8 @@ module.exports = {
     'arrow-parens': 'off',
 
     // import
-    'import/no-extraneous-dependencies': ['warn', { devDependencies: false }],
+    'import/no-extraneous-dependencies': ['error', { devDependencies: true }],
+    // 'import/no-extraneous-dependencies': ['warn', { devDependencies: false }],
```
Comment (iwisunny) [inline .eslintrc.js]: This line can be removed.

File: src/components/Base/CodeMirror/index.jsx [hunk 2]
```diff
@@ -4,10 +4,12 @@ import PropTypes from 'prop-types';
 const CodeMirror = lazy(() => import('react-codemirror'));
 
 // fixme
+/* eslint-disable import/no-extraneous-dependencies */
 if (process.browser) {
   require('codemirror/mode/yaml/yaml');
```
Comment (iwisunny) [inline src/components/Base/CodeMirror/index.jsx]: reduce package deps

File: src/components/Layout/Portal/Dev/index.jsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import classnames from 'classnames';
+/* import PropTypes from 'prop-types';
```
Comment (iwisunny) [inline src/components/Layout/Portal/Dev/index.jsx]: Comment lines can be removed

File: src/components/Layout/Portal/Normal/index.jsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import classnames from 'classnames';
+/* import PropTypes from 'prop-types';
```
Comment (iwisunny) [inline src/components/Layout/Portal/Normal/index.jsx]: No need comment, just remove it

File: .eslintrc.js (hunk 1 above)
Comment (liiil825) [reply .eslintrc.js]: done

File: src/components/Base/CodeMirror/index.jsx (hunk 2 above)
Comment (liiil825) [reply src/components/Base/CodeMirror/index.jsx]: ok

File: src/components/Layout/Portal/Dev/index.jsx (hunk 3 above)
Comment (liiil825) [reply src/components/Layout/Portal/Dev/index.jsx]: done

File: src/components/Layout/Portal/Normal/index.jsx (hunk 4 above)
Comment (liiil825) [reply src/components/Layout/Portal/Normal/index.jsx]: done

**Pull request merged**